.changelog/3289.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:docs
+Add a note about using `-allowed-redirect-uri` when setting up UI-based OIDC auth
```

Comment:
I don't think we support a `docs` section in the changelog, and this is a website change rather than a change with Waypoint, so I don't think we need this changelog.

Reply:
oh thanks. I'll remove it then